### PR TITLE
Store change history for each published guide with update_type=major

### PR DESCRIPTION
**Pull request by AndrewVos**
https://trello.com/c/qzHbv9S9/181-show-change-notes-and-page-history-at-the-bottom-of-guidance-page

Dependent on:

https://github.com/alphagov/government-frontend/pull/129
https://github.com/alphagov/service-manual-publisher/pull/209
https://github.com/alphagov/govuk-content-schemas/pull/274

### CODE REVIEW

File: app/controllers/guides_controller.rb [hunk 1]
```diff
@@ -147,8 +147,8 @@ def guide_params(with = {})
         :related_discussion_href,
         :related_discussion_title,
         :update_type,
+        :change_reason_for_change,
```
Comment (thehenster) [inline app/controllers/guides_controller.rb]: Ha :)

Maybe it'd be easier if these columns didn't match the schema fields. `change_summary` and `change_description` and we map them in the presenter?

File: app/models/edition.rb [hunk 2]
```diff
@@ -88,6 +89,7 @@ def change_note_html
   def draft_copy
     dup.tap do |e|
       e.change_note = nil
+      e.change_reason_for_change = nil
```
Comment (thehenster) [inline app/models/edition.rb]: I think this method along with #ensure_draft_exists can go can't it? They're no longer used?

File: app/controllers/guides_controller.rb (hunk 1 above)
Comment (thehenster) [inline app/controllers/guides_controller.rb]: Or just name it `editons.reason_for_change`? The `change_reason_for_change` is a bit of a mouthful.

File: app/models/edition.rb (hunk 2 above)
Comment (AndrewVos) [inline app/models/edition.rb]: Not sure?

File: app/controllers/guides_controller.rb (hunk 1 above)
Comment (AndrewVos) [inline app/controllers/guides_controller.rb]: K will go with the latter

File: app/models/edition.rb (hunk 2 above)
Comment (thehenster) [inline app/models/edition.rb]: They're no longer used. You can delete `Guide#ensure_draft_exists` and `Edition#draft_copy` along with tests for both.

Comment (AndrewVos) [inline app/models/edition.rb]: Out of the scope of this change, surely?

Comment (thehenster) [inline app/models/edition.rb]: I'm a liar. `Edition#draft_copy` is used in `Guide#latest_editable_edition`. I'll delete the other method separately.

**Comment by thehenster** [comment]
Looks good to me

**Pull request merged**